### PR TITLE
Testo

### PR DESCRIPTION
**Pull request by Pray4Salt**
Testing git merge

### CODE REVIEW

**Review by Frederik-Lausterer — COMMENTED** [review]
text should be Gold

File: Scenes/World.tscn [hunk 1]
```diff
@@ -48,7 +48,7 @@ margin_left = 446.0
 margin_top = 269.0
 margin_right = 802.0
 margin_bottom = 309.0
-text = "Gold = "
```
Comment (Pray4Salt) [inline Scenes/World.tscn:51]: why is this change necessary wouldn't placeholder be better for understanding purpose
